en, zh: Add description about spec.toolImage (#1146)
pingcap/docs-tidb-operator: en/backup-restore-overview.md

### General fields

* `.spec.metadata.namespace`: The namespace where the `Backup` CR is located.

> * `.spec.toolImage`：set the tools image used by `Backup`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/dumpling:v4.0.10` when using Dumpling. If not set, `pingcap/br:${tikv_version}` will be used for backup with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for backup with Dumpling by default. v1.1.9 and the later versions support this configuration.

Reviewer: ```suggestion
* `.spec.toolImage`: The tool image used by `Backup`.
    - When using BR for backup, you can specify the BR version in this field. For example, `.spec.toolImage: pingcap/br:v4.0.10`. If not specified, `pingcap/br:${tikv_version}` is used for backup by default.
    - When using Dumpling for backup, you can specify the Dumpling version in this field. For example, `spec.toolImage: pingcap/dumpling:v4.0.10`. If not specified, the Dumpling version specified in `TOOLKIT_VERSION` of the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) is used for backup by default.          
    - TiDB Operator supports this configuration starting from v1.1.9.
```

> * `.spec.toolImage`：set the tools image used by `Backup`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/dumpling:v4.0.10` when using Dumpling. If not, BR will use the version in `pingcap/br:${tikv_version}` by default. When using Dumpling to backup, the version specified in `TOOLKIT_V40` stored in [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used by default. v1.1.9 and the later versions support this configuration.

Reviewer 2: ```suggestion
* `.spec.toolImage`：set the tools image used by `Backup`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/dumpling:v4.0.10` when using Dumpling. If not set, `pingcap/br:${tikv_version}` will be used for backup with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for backup with Dumpling by default. v1.1.9 and the later versions support this configuration.
```

> * `.spec.toolImage`：set the tools image used by `Backup`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/dumpling:v4.0.10` when using Dumpling. If not, BR will use the version in `pingcap/br:${tikv_version}` by default。 When using Dumpling to backup, the version specified in `TOOLKIT_V40` stored in [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/${operator_version}/images/tidb-backup-manager/Dockerfile) will be used by default. v1.1.9 and the later versions support this configuration.

Reviewer 2: Please replace the `。`.

Author: fixed

This section introduces the fields in the `Restore` CR.

* `.spec.metadata.namespace`: The namespace where the `Restore` CR is located.

> * `.spec.toolImage`：set the tools image used by `Restore`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/lightning:v4.0.10` when using Lightning. If not set, `pingcap/br:${tikv_version}` will be used for backup with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for backup with Lightning by default. v1.1.9 and the later versions support this configuration.

Reviewer 2: ```suggestion
* `.spec.toolImage`：set the tools image used by `Restore`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/lightning:v4.0.10` when using Lightning. If not set, `pingcap/br:${tikv_version}` will be used for a restore with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for a restore with Lightning by default. v1.1.9 and the later versions support this configuration.
```

Reviewer 2: Please update all the versions to 5.0.1 in master.

Author: ok

> * `.spec.toolImage`：set the tools image used by `Restore`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/lightning:v4.0.10` when using Lightning. If not set, `pingcap/br:${tikv_version}` will be used for a restore with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for a restore with Lightning by default. v1.1.9 and the later versions support this configuration.

Reviewer: ```suggestion
* `.spec.toolImage`：The tools image used by `Restore`. 
    - When using BR for restoring, you can specify the BR version in this field. For example,`spec.toolImage: pingcap/br:v4.0.10`. If not specified, `pingcap/br:${tikv_version}` is used for restoring by default.
    - When using Lightning for restoring, you can specify the Lightning version in this field. For example, `spec.toolImage: pingcap/lightning:v4.0.10`. If not specified, the Lightning version specified in `TOOLKIT_VERSION` of the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) is used for restoring by default. 
    - TiDB Operator supports this configuration starting from v1.1.9. 
```
